usr/share/gamescope-session/device-quirks

```diff
@@ -37,6 +37,34 @@ if [[ ":$OXP_LIST:" =~ ":$SYS_ID:"  ]] || [[  ":$AOK_LIST:" =~ ":$SYS_ID:"   ]];
   export STEAM_DISPLAY_REFRESH_LIMITS=40,60
 fi
 
+# OneXFly
+if [[ ":ONEXPLAYER F1:" =~ ":$SYS_ID:"  ]]; then
```

Comment:
Can we create a new list, OXP_120_LIST, and check against that instead to keep it in line with what we're doing with the other devices please?

Reply:
> Can we create a new list, OXP_120_LIST, and check against that instead to keep it in line with what we're doing with the other devices please?

No problem, I will make a slight modification.

Reply:
Cool. If everything is the same otherwise we can also simplify the blocks and add the fly to both lists, then just check the 120 list for the different export at the end with an if else.